### PR TITLE
adding registry endpoint and domain gateway configuration

### PR DESCRIPTION
**Pull request by Shobhajayanna**
adding registry endpoint and domain gateway configuration

List the file(s) included in this PR:
api-mediation-multi-tenancy.md

### CODE REVIEW

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ components.gateway.apiml.service.additionalRegistration:
                 serviceUrl: /
 ```
 
+```
+components.gateway.apiml.security.x509:
+    #  cloud gateway port 
+        certificatesUrl: https://ca32.lvn.broadcom.net:27558/gateway/certificates
```
Comment (arxioly) [inline docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:44]: Please, remove real host, port and system names from everywhere.

**Review by janan07 — APPROVED** [review]

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md (hunk 1 above)
Comment (janan07) [reply docs/user-guide/api-mediation/api-mediation-multi-tenancy.md]: I think this branch needs to be rebased with docs-staging as there are conflicts.

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md [hunk 2]
```diff
@@ -60,6 +60,13 @@ components.gateway.apiml.service.additionalRegistration:
               - gatewayUrl: /
                 serviceUrl: /
 ```
+
+```
+components.gateway.apiml.security.x509:
+    #  cloud gateway port 
+        certificatesUrl: https://sys1:10018/gateway/certificates
```
Comment (arxioly) [inline docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:67]: I would have it similar like in other articles, w/o specific port and host. Something like this:
```suggestion
        certificatesUrl: https://{cloudGatewayHost}:{cloudGatewayPort}/gateway/certificates
```

Comment (arxioly) [reply docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:67]: Ok, it's an example actually. Then it's fine.

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md [hunk 3]
```diff
@@ -350,11 +363,86 @@ This request lists services in the apimlId domain.
...
+				        },
                 "apiId": [
         "zowe.apiml.gateway"
 ],
                 "serviceId": "gateway"
+            },
+            {
+             "status": "UP",
+                "customMetadata": {
+                 "zos.sysname": "sys1",
```
Comment (arxioly) [inline docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:378]: Maybe different sysnames for different APIMLs? To create better visibility that they are on different systems.

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md [hunk 4]
```diff
@@ -397,7 +485,7 @@ services:
      apiInfo:
          - apiId: zowe.apiml.gateway
            gatewayUrl: api/v1
-           swaggerUrl: https://localhost:10012/discoverableclient/v2/api-docs
+           swaggerUrl: https://sys1:10012/discoverableclient/v2/api-docs
      customMetadata:
          apiml:
              service.apimlId: Gateway-CA32
```
Comment (arxioly) [inline docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:491]: Please rename those apimlId with our system name in it.

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md [hunk 5]
```diff
@@ -350,11 +363,86 @@ This request lists services in the apimlId domain.
...
+				},
+                "apiId": [
+        "zowe.apiml.gateway"
+  ],
+                "serviceId": "gateway"
+            },
+            {
+                "status": "UP",
+                "customMetadata": {
+                 "zos.sysname": "sys1",
```
Comment (arxioly) [inline docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:414]: Just to be consistent...
```suggestion
                 "zos.sysname": "sys2",
	 	 "zos.sysplex": "sysplex"
				},
                "apiId": [
        "zowe.apiml.gateway"
  ],
                "serviceId": "gateway"
            },
            {
                "status": "UP",
                "customMetadata": {
                 "zos.sysname": "sys2",
```

File: docs/user-guide/api-mediation/api-mediation-multi-tenancy.md [hunk 6]
```diff
@@ -424,10 +512,10 @@ services:
      apiInfo:
          - apiId: zowe.apiml.gateway
            gatewayUrl: api/v1
-           swaggerUrl: https://localhost:10012/discoverableclient/v2/api-docs
+           swaggerUrl: https://sys1:{discoverableClientPort}/discoverableclient/v2/api-docs
```
Comment (arxioly) [inline docs/user-guide/api-mediation/api-mediation-multi-tenancy.md:515]: Same here.
```suggestion
           swaggerUrl: https://sys2:{discoverableClientPort}/discoverableclient/v2/api-docs
```

**Review by arxioly — APPROVED** [review]
Thanks or applying all comments

**Pull request merged**